src/gzip.rs

```diff
@@ -192,37 +192,33 @@ impl From<u8> for Flags {
 
 impl fmt::Debug for Flags {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "Flags(")?;
-        let mut first = true;
-        let mut item = |s: &str| {
-            if first {
-                first = false;
-                write!(f, "{}", s)
-            } else {
-                write!(f, ", {}", s)
-            }
-        };
+        let mut t = f.debug_tuple("Flags");
 
         if self.has(Flags::FTEXT) {
-            item("TEXT")?;
+            t.field(&format_args!("TEXT"));
```

Comment:
It's possible to write `t.field(&"TEXT");` as well but since this calls `Debug::fmt("TEXT")` it will end up being `"TEXT"` -- including the quotes.